frontera/core/manager.py

```diff
@@ -391,6 +402,9 @@ def request_error(self, request, error):
         #self.event_log_manager.page_crawled_error(processed_page, error)
         return processed_page
 
+    def make_request(self, url, method='GET', headers=None, cookies=None, meta=None):
```

Comment:
Why is that needed? Request model is accessible in backend, why not to instantiate it directly?